dockers/docker-base-stretch/Dockerfile.j2

```diff
@@ -53,7 +53,11 @@ RUN apt-get update &&        \
         libjemalloc1         \
         liblua5.1-0          \
         lua-bitop            \
-        lua-cjson
+        lua-cjson            \
+# For DHCP Monitor tool
+        libexplain-dev       \
```

Comment:
Why do you need dev package?
I think we're not going build anything here
libexplain51 doesn't depend on it
https://packages.debian.org/stretch/libexplain51

Reply:
ACK. will remove the libexplain-dev.